src/content/blog/astro-330.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: "Astro 3.3: <Picture /> component"
```

Comment:
Not sure if it’s supported, but wondering if `&nbsp;` is usable here as this line break is a bit unfortunate?

<img width="802" alt="image" src="https://github.com/withastro/astro.build/assets/357379/4e5e679d-5d15-48c6-9730-cb891358afa0">

<img width="750" alt="image" src="https://github.com/withastro/astro.build/assets/357379/479ba539-7665-4f44-a6d6-f66170240e2d">


If not, perhaps we don’t need the HTML tag style for the title?

```suggestion
title: "Astro 3.3: Picture component"
```

(Haven’t tested but may also be more accessible? Guess the current version could be read out as “less than Picture slash greater than component?)

Reply:
Yep, good call!